### ISSUE TITLE
Storage diagrams to handle User Defined Value Types when introduced in Solidity

### ISSUE THREAD

**Issue by naddison36** [naddison36/sol2uml#139]
sol2uml storage will incorrectly size aliases types to 20 bytes as it'll think it is referencing a contract. This is incorrect if types like uint64 are aliased.

https://docs.soliditylang.org/en/v0.8.18/types.html#user-defined-value-types
https://blog.soliditylang.org/2021/09/27/user-defined-value-types/
https://github.com/ethereum/solidity/issues/11969

**Comment by naddison36** [comment]
Solidity has now released user defined value types in 0.8.19 https://blog.soliditylang.org/2023/02/22/solidity-0.8.19-release-announcement/

**Comment by rori4** [comment]
any update on this?

**Comment by naddison36** [comment]
No, but thanks for asking as I'll spend time on it if there's demand for it.

**Comment by rori4** [comment]
it would be great since that feature is used more and more nowadays. If you need some help with it let me know
